app/scripts/modules/cloudfoundry/src/pipeline/stages/deleteServiceKey/cloudfoundryDeleteServiceKeyStage.module.ts

```diff
@@ -0,0 +1,21 @@
+import { CloudfoundryDeleteServiceKeyStageConfig } from './CloudfoundryDeleteServiceKeyStageConfig';
+import { ExecutionDetailsTasks, IStage, Registry } from '@spinnaker/core';
+import { CloudfoundryServiceKeyExecutionDetails } from 'cloudfoundry/presentation';
+
+Registry.pipeline.registerStage({
+  accountExtractor: (stage: IStage) => stage.context.credentials,
+  configAccountExtractor: (stage: IStage) => [stage.credentials],
+  cloudProvider: 'cloudfoundry',
+  component: CloudfoundryDeleteServiceKeyStageConfig,
+  controller: 'BaseProviderStageCtrl as baseProviderStageCtrl',
```

Comment:
Can we strike `controller` since this is a React stage?

Reply:
👍